### ISSUE TITLE
AZ Navbar Fullscreen (mobile): Confirm focus-visible and active button styles

### ISSUE THREAD

**Issue by bberndt-uaz** [az-digital/arizona-bootstrap#2160]
The "Back to..." button on secondary and tertiary menu pages has a focus ring which is cut off on the left side. The active state for this buttons and the other buttons in the mobile display should be reviewed and corrected if needed.

<img width="50%" alt="Image" src="https://github.com/user-attachments/assets/5fc43878-c713-4c44-a2fc-2d2bcff7d7d0" />

## Conditions of satisfaction
- [x] Fix focus-visible style of "Back to..." button on secondary and tertiary menu pages
- [x] Examine other focus-visible button styles in the mobile display and fix as needed
- [x] Examine active styles in the mobile display and fix as needed
